src/main/java/seedu/address/MainApp.java

```diff
@@ -52,8 +46,8 @@ public void init() throws Exception {
 
         UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
-        // AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
-        // storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
+        storage = new StorageManager(addressBookStorage, userPrefsStorage);
```

Comment:
Should we start renaming the other classes soon?

Reply:
We can discuss the naming during this week's meeting, cause quite a lot will change and I am afraid it might get confusing.

Reply:
alright cools